pkg/services/ngalert/api/api_testing.go

```diff
@@ -111,9 +110,9 @@ func (srv TestingApiSrv) RouteEvalQueries(c *models.ReqContext, cmd apimodels.Ev
 		return ErrResp(http.StatusUnauthorized, fmt.Errorf("%w to query one or many data sources used by the rule", ErrAuthorization), "")
 	}
 
-	if _, err := validateQueriesAndExpressions(c.Req.Context(), cmd.Data, c.SignedInUser, c.SkipCache, srv.DatasourceCache); err != nil {
-		return ErrResp(http.StatusBadRequest, err, "invalid queries or expressions")
-	}
+	// if _, err := validateQueriesAndExpressions(c.Req.Context(), cmd.Data, c.SignedInUser, c.SkipCache, srv.DatasourceCache); err != nil {
+	// 	return ErrResp(http.StatusBadRequest, err, "invalid queries or expressions")
+	// }
```

Comment:
few commented code lines to remove

Reply:
removed those lines as the validation will be performed by the QueriesAndExpressionsEval